### PR TITLE
Add inconsistency

### PR DESCRIPTION
**Pull request by Thiago-NovaesB**
- Update SDDP.jl 1.1 => 1.6
- Changing some functions names to adapt to the Julia convention
- Identification when forward and backward are configured differently and trains SDDP correctly

Starts: https://github.com/LAMPSPUC/HydroPowerModels.jl/issues/7

### CODE REVIEW

**Comment by andrewrosemberg** [comment]
I have added the CI to the github actions. Please rebase to master so that we can see CI in this PR.

**Comment by andrewrosemberg** [comment]
Add inconsistency tests to check if the code is running correctly for the new API (When you have different foward and backward models).

File: Project.toml
Comment (andrewrosemberg) [inline Project.toml:1]: Bump version to show new functionality: `version = "0.1.0"` => `version = "0.2.0"`

Comment (Thiago-NovaesB) [reply Project.toml:1]: done

**Comment by Thiago-NovaesB** [comment]
I added case3_inconsistency based on case3

**Comment by andrewrosemberg** [comment]
@joaquimg @odow Adding the FAST Inconsistent policy training.

**Review by andrewrosemberg — APPROVED** [review]
LGTM! This is great and well executed. Thank you.

**Comment by odow** [comment]
> Adding the FAST

What is "FAST"? Or is this just emphasis on _fast_ in that it runs better now?

**Comment by andrewrosemberg** [comment]
> > Adding the FAST
> 
> What is "FAST"? Or is this just emphasis on _fast_ in that it runs better now?

Sorry, It was how they coined the SDDP with different forward and backward models in the original paper. I normally don't refer to it by FAST but this time I did for some reason haha.

**Pull request merged**